src/Dev/SapphireTest.php

```diff
@@ -1239,6 +1239,15 @@ protected function mockSleep(int $seconds): DBDatetime
      */
     private function setI18nLocale(): void
     {
+        // Skip setting the locale for excluded test classes that are known to cause a
+        // LogicException when calling getCurrentRelativePath(), and we can be sure
```

Comment:
```suggestion
        // Skip setting the locale for excluded test classes that are known to cause an
        // exception when calling getCurrentRelativePath(), and we can be sure
```
If we're doing it with an array under the assumption we may add classes to this list (which I think is unlikely, I still say `instanceof` check is better) - then we shouldn't assume they'll all be `LogicException`

Reply:
Updated